rstudio/rstudio
# Fix accessible keyboard shortcut help for Electron

### Intent

Addresses #10947

### Approach

Implement the desktop callback used by this feature.

### Automated Tests

No tests.

### QA Notes

Verify parity with Qt desktop for Help / Keyboard Shortcuts Help:

- Help / Keyboard Shortcuts Help command displays keyboard.htm in web browser (when screen-reader accessibility mode is enable), and the semi-transparent popup-help pane when accessibility isn't enabled.
- the "See All Shortcuts..." link in upper-right of the semi-transparent popup-help pane launches keyboard.htm in web browser

When you toggle the screen-reader mode (Help / Accessibility / Screen Reader Support) in Electron, you must manually exit and restart RStudio due to issue #10748

### Checklist
